src/softsynth.c

```diff
@@ -64,6 +64,39 @@ static void queue_add_text(char *txt, size_t length)
 	struct espeak_entry_t *entry;
 	int added = 0;
 
+	// code to scan for and undo the character scramblings of Speakup
+	// much easier solution than trying to fix it in Speakup kernel code
```

Comment:
Papering over bugs means we'll continue piling dirt over dirt over dirt... Let's just really fix bugs...